distributed/protocol/tests/test_highlevelgraph.py

```diff
@@ -1,6 +1,7 @@
 import ast
 
 import pytest
+from numpy.testing import assert_array_equal
```

Comment:
This needs to come after `pytest.importorskip("numpy")`

Reply:
Actually `assert_array_equal` is only used in a single test. Let's just move the import into that test, or use the fully qualified `np.testing.assert_array_equal` version